tcms/tests/__init__.py

```diff
@@ -229,13 +229,15 @@ def setUpTestData(cls):
         cls.execution_6 = executions[5]
 
 
-class AssignPermissionsToTestUser(LoggedInTestCase):
-    """Assign required permissions to user """
+class PermissionsTestCase(LoggedInTestCase):
+    """Class is used for assigning permissions on test users. """
```

Comment:
No. This will be a base class for implementing all tests that have to do with permissions. Assigning the correct parmission labels to the test user is just a precondition of being able to exercise and validate the function under test, not the main thing that we do.